content/doc/book/blueocean/index.adoc

```diff
@@ -134,67 +105,51 @@ endif::[]
 
 === Where is the name from?
 
-The name Blue Ocean comes from the book
-link:https://en.wikipedia.org/wiki/Blue_Ocean_Strategy[Blue Ocean Strategy]
-where instead of looking at strategic problems within a contested space, you
-look at problems in the larger uncontested space. To put this more simply,
-consider this quote from ice hockey legend Wayne Gretzky: "skate to where the
-puck is going to be, not where it has been".
+The name Blue Ocean comes from the book link:https://en.wikipedia.org/wiki/Blue_Ocean_Strategy[Blue Ocean Strategy]. 
+This strategy involves looking at problems in the larger uncontested space, instead of strategic problems within a contested space.
+To put this more simply, consider this quote from ice hockey legend Wayne Gretzky: "Skate to where the puck is going to be, not where it has been."
 
 
 ==== Does Blue Ocean support freestyle jobs?
 
-Blue Ocean aims to deliver a great experience around Pipeline and be compatible
-with any freestyle jobs you already have configured on your Jenkins instance.
-However, you will not benefit from any of the features built for Pipelines – for
-example, Pipeline visualization.
+Blue Ocean aims to deliver a great experience around Pipeline, and compatiblity with any freestyle jobs you already have configured in your Jenkins instance.
+However, you will not benefit from the features built for Pipelines, for example Pipeline visualization.
 
-As Blue Ocean is designed to be extensible, it is possible for the Jenkins
-community to extend Blue Ocean to support other job types in the future.
+Blue Ocean was designed to be extensible, so that the Jenkins community could extend Blue Ocean functionality.
+While there will not be any further functionalities added to Blue Ocean, it still provides Pipeline visualization and other features that users find valuable.
 
 
 === What does this mean for the Jenkins classic UI?
 
-The intention is that as Blue Ocean matures, there will be fewer reasons for
-users to go back to the existing "classic UI". Read more about the classic UI
-in link:../pipeline/getting-started[Getting started with Pipeline].
-
+The intention was originally as Blue Ocean matured, there would be fewer reasons for users to go back to the existing classic UI.
+Refer to the link:../pipeline/getting-started[getting started with Pipeline] page for more information about the classic UI.
 For example, early versions of Blue Ocean are mainly targeted at Pipeline jobs.
```

Comment:
Let's admit that the freestyle experience is not significantly improved by Blue Ocean.
```suggestion
Blue Ocean focuses its improvements on Pipeline jobs.
```